.eslintrc.json

```diff
@@ -39,6 +39,12 @@
         "ignoreStrings": true
       }
     ],
+    "prettier/prettier": [
```

Comment:
why?

Reply:
cai nay sua loi end of line o eslint a ah